decidim-proposals/app/events/decidim/proposals/publish_proposal_event.rb

```diff
@@ -35,6 +37,12 @@ def safe_resource_translated_text
         I18n.with_locale(I18n.locale) { translated_attribute(resource_text, nil, true).to_s.html_safe }
       end
 
+      def notification_title
+        i18n_key = author.present? ? "notification_title" : "notification_title_official"
```

Comment:
We also have `Decidim::Proposals::Proposal#official?` so I was wondering why you didn't use that?

Reply:
Sorry, I missed your comment! You made a good point, thanks for the suggestion